web/auth.go

```diff
@@ -17,6 +17,38 @@ import (
 	"golang.org/x/oauth2/github"
 )
 
+// TODO: move/rewrite it when we have other endpoints
+
+type successResp struct {
+	Data interface{} `json:"data"`
+}
+type errorResp struct {
+	Errors []error `json:"errors"`
+}
+
+func successJSON(w http.ResponseWriter, r *http.Request, data interface{}) {
+	b, err := json.Marshal(successResp{data})
+	if err != nil {
+		lg.RequestLog(r).Warn(err.Error())
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	w.Write(b)
+}
+
+func errorJSON(w http.ResponseWriter, r *http.Request, code int, errors ...error) {
```

Comment:
Why is this defined, but then `http.Error` is used?

Reply:
just for testing. As I mentioned in PR description: "This PR doesn't cover proper serialization of responses"